src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ImporterService.cs

```diff
@@ -281,7 +286,8 @@ int rowsPerBatch
             {
                 Id = observationId,
                 FilterItems = GetFilterItems(context, rowValues, colValues, subjectMeta.Filters, observationId),
-                GeographicLevel = GetGeographicLevel(rowValues, colValues),
+                // TODO should be able to drop GeographicLevel in future once the migration to Location is complete
```

Comment:
Is it worth bumping a ticket ref on this if we have one available?

Reply:
I removed this TODO as plan to keep GeographicLevel on Observation for now but raised EES-3067 and EES-3068 and left them as comments in the appropriate place as part of EES-3011.